lib/biokbase/catalog/registrar.py

```diff
@@ -252,7 +252,7 @@ def sanity_checks_and_parse(self, basedir, git_commit_hash):
         with codecs.open(os.path.join(basedir, yaml_filename), 'r', "utf-8",
                          errors='ignore') as kb_yaml_file:
             kb_yaml_string = kb_yaml_file.read()
-        self.kb_yaml = yaml.load(kb_yaml_string)
+        self.kb_yaml = yaml.load(kb_yaml_string, Loader=yaml.FullLoader)
```

Comment:
Does this have any security concerns? IIRC there's a safe loader that is less subject to problems

Reply:
I'm not sure why it's needed, the kbase.yml file is very simple

Reply:
TypeError: load() missing 1 required positional argument: 'Loader'
BUILD_ERROR: load() missing 1 required positional argument: 'Loader'

https://github.com/kbase/catalog/actions/runs/12307585559/job/34351499135?pr=136

Reply:
The question isn't whether there's a loader specified at all, it's whether it needs to be a FullLoader

Reply:
👍

Reply:
:+1: